templates/templates/preview/test.template

```diff
@@ -0,0 +1,117 @@
+{% for model in manifest.models %}
+{% if model is defined %}
+== Models ==  
```

Comment:
There appear to be no test templates for HTML output. Are we not testing HTML/file-based output?

Reply:
removed

Reply:
added html output templates